### PR TITLE
Apply phpstan fixes

### PR DESCRIPTION
**Pull request by greg0ire**
I am targetting this branch, because this is BC (and does not warrant a release.

# Subject

I'm trying to run phpstan/phpstan on the admin bundle.

### CODE REVIEW

File: Controller/HelperController.php [hunk 1]
```diff
@@ -444,7 +443,7 @@ public function retrieveAutocompleteItemsAction(Request $request)
      * @param AdminInterface $admin
      * @param string         $field
      *
-     * @return FormInterface
+     * @return \Sonata\AdminBundle\Admin\FieldDescriptionInterface
```
Comment (OskarStark) [inline Controller/HelperController.php]: I am 👎  for this and keep the use statement

File: Form/ChoiceList/ModelChoiceLoader.php [hunk 2]
```diff
@@ -26,6 +26,8 @@
  */
 class ModelChoiceLoader implements ChoiceLoaderInterface
 {
+    public $identifier;
```
Comment (OskarStark) [inline Form/ChoiceList/ModelChoiceLoader.php]: a public var should have a PHP Doc

File: Controller/HelperController.php (hunk 1 above)
Comment (greg0ire) [inline Controller/HelperController.php]: Ok, but the main change was about `FormInterface` => `FieldDescriptionInterface`

Comment (OskarStark) [inline Controller/HelperController.php]: yeah, that is absolutely fine!

File: Form/ChoiceList/ModelChoiceLoader.php (hunk 2 above)
Comment (greg0ire) [inline Form/ChoiceList/ModelChoiceLoader.php]: I don't know what this is supposed to be, and it's not meant to stay public forever BTW.

**Pull request merged**